docs/userGuide/syntax/mathformulae.mbdf

```diff
@@ -0,0 +1,60 @@
+## Math Formulae
+Markbind supports typesetting math equations with [KaTeX](https://katex.org).
+
+<box type="info">
+
+Insert inline equations by enclosing them in round brackets `\( ... \)`.
+
+Insert block equations by enclosing them in square brackets `\[ ... \]`.
+
+Insert numbered block equations by enclosing the equation square brackets and the equation number in curly brackets `\[ ... \] (1)`.
+</box>
+
+<box type="important">
+
+You may leave whitespace between the brackets and your equation.
+
+Correct: `\( 1+1=2 \)`, Correct: `\(1+1=2\)`
+
+Ensure that there is a newline between an inline equation and a block equation, or formatting will not work.
```

Comment:
it should be not be preceded by a markdown paragraph.
Should we remove this since its a markdown spec thing instead? Then enhance the examples further below to make the use case of inline / display equations clearer instead

e.g. this does not work as well
```
abc
\[2+2=4\]

or this 

> abc
> \[2+2=4\]
```